docs/content/foundations/layout-positioning/display.mdx

```diff
@@ -10,33 +10,135 @@ utility_modules:
     utility_module_name: 'display'
     utility_class_intro: |
       The display utility classes have not been modified from the U.S. Web Design System (USWDS). Refer to the [USWDS Display utility page](https://designsystem.digital.gov/utilities/display/#utility-display) for a complete listing of classes.
-    utility_class_display_component: 'TestUtilityClassTable'
+    utility_class_display_component: 'DisplayDemonstration'
     utility_class_display_params:
-      classes: []
+      type: 'display'
+      classes:
+        - class_name: 'display-block'
+          calculated_value: 'block'
+        - class_name: 'display-flex'
+          calculated_value: 'flex'
+        - class_name: 'display-inline'
+          calculated_value: 'inline'
+        - class_name: 'display-inline-block'
+          calculated_value: 'inline-block'
+        - class_name: 'display-inlne-flex'
+          calculated_value: 'inlne-flex'
+        - class_name: 'display-none'
+          calculated_value: 'none'
+        - class_name: 'display-table-cell'
+          calculated_value: 'table-cell'
+    utility_examples:
+      - heading:
+        description: |
+        preview: false
+        code: |
+          <section>
```

Comment:
A section element is only allowed if there is a heading. There is no heading, remove this. It doesn't appear a wrapper element is needed for this anyway.